Source/Shaders/Builtin/Structs/materialInput.glsl

```diff
@@ -10,6 +10,8 @@
  * @property {vec3} normalEC Unperturbed surface normal in eye coordinates.
  * @property {mat3} tangentToEyeMatrix Matrix for converting a tangent space normal to eye space.
  * @property {vec3} positionToEyeEC Vector from the fragment to the eye in eye coordinates.  The magnitude is the distance in meters from the fragment to the eye.
+ * @property {float} height The height of the terrain.  Only available for globe materials.
```

Comment:
Can this be more explicit?  I assume the height in meters above or below the WGS84 ellipsoid?